test/k8s-integration/main.go

```diff
@@ -231,13 +248,24 @@ func setEnvProject(project string) error {
 	return nil
 }
 
-func runTests(pkgDir, k8sDir, storageClassFile, testFocus string) error {
-	testDriverConfigFile, err := generateDriverConfigFile(pkgDir, storageClassFile)
-	if err != nil {
-		return err
+func runTests(pkgDir, k8sDir, storageClassFile, testFocus, gceZone string, migrationTest bool) error {
```

Comment:
since these two tests are mutually exclusive, should we have two functions? One for setting the appropriate arguments for each?  And they can funnel down to the common method.